src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -439,6 +439,27 @@
    "CREATE INDEX catalogs_hash_expr_idx ON catalogs(trim(leading '\\x' from hash::text))"
    "CREATE INDEX catalogs_certname_idx ON catalogs (certname)"))
 
+(defn add-indexes-for-reports-summary-query
```

Comment:
Is there something we can do to test this in the PuppetDB repo? Best I can tell right now we have to go a decent distance before we'd find that this is broken.